From ccff091f86d1f43e22f7f7e71194c60464d5c992 Mon Sep 17 00:00:00 2001
From: Martin Petrov <martin@martinpetrov.com>
Date: Tue, 8 Nov 2011 14:47:13 -0500
Subject: [PATCH] modified snipmate to use leader key

---
 after/plugin/snipMate.vim |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/after/plugin/snipMate.vim b/after/plugin/snipMate.vim
index bdbe199..f13bd35 100644
--- a/after/plugin/snipMate.vim
+++ b/after/plugin/snipMate.vim
@@ -10,8 +10,8 @@ let s:did_snips_mappings = 1
 "
 " You can safely adjust these mappings to your preferences (as explained in
 " :help snipMate-remap).
-ino <silent> <tab> <c-r>=TriggerSnippet()<cr>
-snor <silent> <tab> <esc>i<right><c-r>=TriggerSnippet()<cr>
+ino <silent> <leader><tab> <c-r>=TriggerSnippet()<cr>
+snor <silent> <leader> <tab> <esc>i<right><c-r>=TriggerSnippet()<cr>
 ino <silent> <s-tab> <c-r>=BackwardsSnippet()<cr>
 snor <silent> <s-tab> <esc>i<right><c-r>=BackwardsSnippet()<cr>
 ino <silent> <c-r><tab> <c-r>=ShowAvailableSnips()<cr>
-- 
1.7.5.4

